src/sentry/static/sentry/app/views/eventsV2/table/columnEditRow.tsx

```diff
@@ -1,66 +1,101 @@
 import React from 'react';
 import styled from '@emotion/styled';
+import {components} from 'react-select';
 
+import Badge from 'app/components/badge';
 import SelectControl from 'app/components/forms/selectControl';
-import {OrganizationSummary, SelectValue} from 'app/types';
+import {SelectValue} from 'app/types';
+import {t} from 'app/locale';
 import space from 'app/styles/space';
 
-import {
-  AGGREGATIONS,
-  FIELDS,
-  TRACING_FIELDS,
-  Field,
-  Aggregation,
-} from '../eventQueryParams';
+import {ColumnType, AggregateParameter} from '../eventQueryParams';
 import {Column} from '../eventView';
-import {AGGREGATE_ALIASES} from '../data';
+
+export enum FieldValueKind {
+  TAG = 'tag',
+  FIELD = 'field',
+  FUNCTION = 'function',
+}
+
+// Payload of select options used to update column
+// data as the first picker has tags, fields and functions all combined.
+export type FieldValue =
```

Comment:
Would these be better in a `types` file?

Reply:
Good idea. I'll move all the types in a follow up pull request.